benefits/core/templates/core/help.html

```diff
@@ -3,65 +3,138 @@
 {% load static %}
 
 {% block page-title %}
-  {% translate "core.buttons.help" %}
+  {% translate "Help" %}
 {% endblock page-title %}
 
 {% block main-content %}
   <div class="container">
     <div class="row justify-content-center">
       <div class="col-md-8 col-10">
-        <h1 class="text-center">{% translate "core.buttons.help" %}</h1>
+        <h1 class="text-center">{% translate "Help" %}</h1>
       </div>
     </div>
 
     <div class="row justify-content-center">
       <div class="col-md-8 col-10">
-        <h2 class="h2-sm pt-4 pt-lg-8" id="what-is-cal-itp">{% translate "core.pages.help.about" %}</h2>
-        <p class="pt-2 pt-lg-4">{% translate "core.pages.help.about.p[0]" %}</p>
-        <p class="pt-3 pt-lg-4">{% translate "core.pages.help.about.p[1]" %}</p>
+        <h2 class="h2-sm pt-4 pt-lg-8" id="what-is-cal-itp">{% translate "What is Cal-ITP Benefits?" %}</h2>
+        <p class="pt-2 pt-lg-4">
+          {% blocktranslate trimmed %}
+            The Benefits tool is provided by California Integrated Travel Project (Cal-ITP), which is a new program from the
+            California Department of Transportation dedicated to making travel simpler and cost-effective for all.
+          {% endblocktranslate %}
+        </p>
+        <p class="pt-3 pt-lg-4">
+          {% blocktranslate trimmed %}
+            We partner with local transit agencies, the California Department of Technology, and Login.gov to confirm the
+            identity of riders who are eligible to receive transit benefits. Once we are able to confirm a rider's identity and
```

Comment:
rider’s

Reply:
This is a plural noun right? Not possessive here. 

e.g. "There are many riders that may be eligible"

Reply:
86a0213cb53844cc59db47450c8f3c7fde86c826